README.md

```diff
@@ -1 +1,38 @@
-# Enumerable
\ No newline at end of file
+# Enumerable
+# Decoder
+This is project of creating customized Enurables  using Ruby.
+
+## Author 1
+
+👤 **Mustafa Fahimy**
+
+- GitHub: [@fahimy143](https://github.com/fahimy143)
+- LinkedIn: [Mustafa Fahimy](https://www.linkedin.com/in/mustafa-fahimy-307566236/)
```

Comment:
- [ ] For your README, please use this [template](https://github.com/microverseinc/readme-template) as a reference and ensure that you have all the main headers so that your project is more professional👍